gems/bootstrap/CVE-2018-14040.yml

```diff
@@ -0,0 +1,25 @@
+---
+gem: bootstrap
+cve: 2018-14040
+url: https://blog.getbootstrap.com/2018/07/12/bootstrap-4-1-2/
+title: XSS vulnerabilities in bootstrap
```

Comment:
add "via data-parent, data-target, data-container" or via "data-*" properties or something? Just to help differentiate.